### PR TITLE
[docs] PR for Document the new cluster aware jdbc driver (Doc9156)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Doc build preview: https://deploy-preview-9240--infallible-bardeen-164bc9.netlify.app/latest/integrations/smart-driver/

### CODE REVIEW

File: docs/content/latest/integrations/smart-driver.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+showAsideToc: true
+---
+
+JDBC Smart Driver for YSQL is a distributed driver based on the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+
+The following are the key features of JDBC Smart Driver:
+
+- It is **cluster-aware**, which eliminates the need for an external load balancer.
+
+  The driver package includes a `YBClusterAwareDataSource` class that uses one initial contact point for the YugabyteDB cluster as a means of discovering all the nodes and automatically gathering information about the state of each node (added, started, stopped, removed).
```
[the remark this answers is not in the document]
Comment (kneeraj) [reply docs/content/latest/integrations/smart-driver.md:21]: Pleaes shorten it but I the meaning should be clear now I guess.

File: docs/content/latest/integrations/smart-driver.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+1. Clone the following repository:
+
+   ```sh
+   git clone https://github.com/yugabyte/jdbc-yugabytedb.git && cd jdbc-yugabytedb
+   ```
+
+2. Build and install into your local Maven folder by executing the following command:
+
+   ```sh
+   mvn clean install -DskipTests
```
Comment (kneeraj) [inline docs/content/latest/integrations/smart-driver.md:56]: If gpg key error comes you can avoid it by instead using this command:

`mvn clean install -DskipTests -Dgpg.skip`

File: docs/content/latest/integrations/smart-driver.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+   ```
+
+3. Add the following lines to your Maven project:
+
+   ```properties
+   <dependency>
+       <groupId>com.yugabyte</groupId>
+       <artifactId>jdbc-yugabytedb</artifactId>
+       <version>42.2.7-yb-5-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (kneeraj) [reply docs/content/latest/integrations/smart-driver.md:65]: Change this "42.2.7-yb-5-SNAPSHOT" to "42.2.7-yb-5-beta.1"

File: docs/content/latest/integrations/smart-driver.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+   </dependency> 
+   ```
+
+## Using the Smart Driver
+
+Load balancing is performed using the following connection properties:
```
Comment (kneeraj) [inline docs/content/latest/integrations/smart-driver.md:71]: We need to segregate the cluster aware load balancing with the topology aware.
For cluster aware only 'load-balance=true' is required

and for topology aware in addition to the above property even the second property is required which is 'topology-keys'

File: docs/content/latest/integrations/smart-driver.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+  - *[-v] [-i]* &nbsp; with `-v` `-i` if you want to run the script in both `VERBOSE` and `INTERACTIVE` mode at the same time.
+
+  - *</path_to_yugabyte_installation/>* &nbsp; with the path to the directory where you installed YugabyteDB. 
+
+  <br>For more details, you can consult the help for the script.<br>
+
+  <br> The following is an example of a shell command that runs the script:
+
+  ```sh
+  ./run.sh -v -i -D ~/yugabyte-2.7.1.1/
```
Comment (kneeraj) [inline docs/content/latest/integrations/smart-driver.md:169]: This version, 2.7.1.1 is old. And we cannot provide a version yet as it is not pub;ished yet. In the works. Will keep you posted about this.

File: docs/content/latest/integrations/smart-driver.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+### How to Obtain the Driver from Maven
+
+To get the driver from Maven, add the following lines to your Maven project:
+
+```properties
+<dependency>
+  <groupId>com.yugabyte</groupId>
+  <artifactId>jdbc-yugabytedb</artifactId>
+  <version>42.2.7-yb-5</version>
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:39]: Changed

File: docs/content/latest/integrations/smart-driver.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:65]: Changed

File: docs/content/latest/integrations/smart-driver.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:21]: Changed

File: docs/content/latest/integrations/smart-driver.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:71]: Fixed

File: docs/content/latest/integrations/smart-driver.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:169]: Okay

File: docs/content/latest/integrations/smart-driver.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:56]: Added

**Comment by lizayugabyte** [comment]
@kneeraj, I implemented the review comments. Could you, please, take a quick look and let me know if the doc is good to go?

File: docs/content/latest/integrations/smart-driver.md (hunk 5 above)
Comment (kneeraj) [reply docs/content/latest/integrations/smart-driver.md:178]: 2.7.1.1 is still there. We need to fix it and for that we need to know the exact version which we don't have right now. Will talk to @rao-vasireddy and get back

Comment (lizayugabyte) [reply docs/content/latest/integrations/smart-driver.md:178]: We can replace the version with the user-defined version for now:
`./run.sh -v -i -D ~/yugabyte-<yugabytedb-version>/`

You need to replace _yugabytedb-version_ with the version of YugabyteDB you are using. (or something like that)

Then we'll edit the doc when the actual version is available.

**Review by kneeraj — APPROVED** [review]

**Pull request merged**